apps/pie-docs/src/patterns/cookie-banner/code/props.json

```diff
@@ -0,0 +1,59 @@
+{
+    "headings": ["Prop", "Options", "Description", "Default"],
+    "rows": [
+        [
+            "hasPrimaryActionsOnly",
+            {
+                "type": "code",
+                "item": ["true", "false"]
+            },
+            "When true, sets the variant to \"primary\" for the button which accepts necessary cookies only.",
+            {
+                "type": "code",
+                "item": ["false"]
+            }
+        ],
+        [
+            "locale",
+            "Any locale from [this list](https://github.com/justeattakeaway/pie/tree/main/packages/components/pie-cookie-banner/locales)",
+            "Assigns the localisation data for the component strings.",
+            "[English language locale](https://github.com/justeattakeaway/pie/blob/d7798b2b406e22618ffc2ae4e372c2fb5c41f187/packages/components/pie-cookie-banner/src/index.ts#L33)"
```

Comment:
Why do we link to a specific commit here?

Reply:
I wanted to point to a code example, but if it is confusing I will just leave the text instead of having the link :) 
If we want to link specific code examples in the future, should we just link to the files? or is there any other way to get to a specific point in the code?